spec/lib/salus/report_spec.rb

```diff
@@ -178,6 +178,35 @@ def build_report(report_uri)
       end
     end
 
+    context 'HTTP report URI given with request parameters' do
+      it 'should make a call to send the report for http URI' do
```

Comment:
This spec looks good.  It's testing format: json. We should probably copy and make a test for txt, yaml, sarif too